src/vcpkg-test/arguments.cpp

```diff
@@ -160,3 +160,24 @@ TEST_CASE ("Feature flag off", "[arguments]")
     auto v = VcpkgCmdArguments::create_from_arg_sequence(t.data(), t.data() + t.size());
     CHECK(!v.versions_enabled());
 }
+
+TEST_CASE ("CMake debugger flags", "[arguments]")
+{
+    std::vector<std::string> t = {"--x-cmake-debug",
+                                  "\\\\.\\pipe\\tespipe;zlib;bar;baz",
```

Comment:
I really would like to pass `\\.\pipe\testpipe` on the command line instead of having to escape the `\`. Is there some trick to achieve that? Who actually eats the `\` ? Is it the command line or is it done internally in vcpkg somehow?

Reply:
The C++ compiler is eating the `\`s. Raw literals could work?

Reply:
The compiler cannot eat on the command line since it already did it business (speaking about compiled vcpkg.exe not the test). I would like to transform:
`./vcpkg.exe install zlib --x-cmake-configure-debug \\\\.\\pipe\\tespipe`
into 
`./vcpkg.exe install zlib --x-cmake-configure-debug \\.\pipe\tespipe`

Reply:
> The compiler cannot eat on the command line since it already did it business (speaking about compiled vcpkg.exe not the test)

Then I'm not sure what you're referring to here:

```console
PS C:\Dev\vcpkg-tool\out\build\Win-x64-Debug-WithArtifacts> .\vcpkg.exe install zlib:x64-windows --debug --x-cmake-debug  \\.\pipe\tespipe`;zlib
[DEBUG] To include the environment variables in debug output, pass --debug-env
[DEBUG] Trying to load bundleconfig from C:\Dev\vcpkg-tool\out\build\Win-x64-Debug-WithArtifacts\vcpkg-bundle.json
[DEBUG] Failed to open: C:\Dev\vcpkg-tool\out\build\Win-x64-Debug-WithArtifacts\vcpkg-bundle.json
[DEBUG] Bundle config: readonly=false, usegitregistry=false, embeddedsha=nullopt, deployment=Git, vsversion=nullopt
[DEBUG] Metrics enabled.
[DEBUG] Feature flag 'binarycaching' unset
[DEBUG] Feature flag 'compilertracking' unset
[DEBUG] Feature flag 'registries' unset
[DEBUG] Feature flag 'versions' unset
[DEBUG] Feature flag 'dependencygraph' unset
[DEBUG] Using scripts-root: C:\Dev\vcpkg\scripts
[DEBUG] Using builtin-ports: C:\Dev\vcpkg\ports
[DEBUG] Using installed-root: C:\Dev\vcpkg\installed
[DEBUG] Using buildtrees-root: C:\Dev\vcpkg\buildtrees
[DEBUG] Using packages-root: C:\Dev\vcpkg\packages
[DEBUG] Using vcpkg-root: C:\Dev\vcpkg
[DEBUG] Using scripts-root: C:\Dev\vcpkg\scripts
[DEBUG] Using builtin-registry: C:\Dev\vcpkg\versions
[DEBUG] Using downloads-root: C:\Dev\vcpkg-downloads
[DEBUG] Detecting host with IsWow64Process2
Computing installation plan...
[DEBUG] Failed to open: C:\Dev\vcpkg\ports\llvm-lto-lib\vcpkg.json
[DEBUG] Found path: C:\Program Files\CMake\bin\cmake.exe
[DEBUG] 1000: CreateProcessW("C:\Dev\vcpkg-downloads\tools\cmake-3.27.1-windows\cmake-3.27.1-windows-i386/bin/cmake.exe" --version)
[DEBUG] 1000: cmd_execute_and_stream_data() returned 0 after   109348 us
[DEBUG] 1001: CreateProcessW("C:\Dev\vcpkg-downloads\tools\cmake-3.27.1-windows\cmake-3.27.1-windows-i386/bin/cmake.exe" -DVCPKG_ROOT_DIR=C:/Dev/vcpkg -DPACKAGES_DIR=C:/Dev/vcpkg/packages -DBUILDTREES_DIR=C:/Dev/vcpkg/buildtrees -D_VCPKG_INSTALLED_DIR=C:/Dev/vcpkg/installed -DDOWNLOADS=C:/Dev/vcpkg-downloads -DVCPKG_MANIFEST_INSTALL=OFF -P "C:\Dev\vcpkg\buildtrees\0.vcpkg_tags.cmake")
[DEBUG] 1001: cmd_execute_and_stream_data() returned 0 after    48314 us
The following packages will be built and installed:
    zlib:x64-windows -> 1.2.13
[DEBUG] 1002: CreateProcessW("C:\Program Files (x86)\Microsoft Visual Studio\Installer\vswhere.exe" -all -prerelease -legacy -products * -format xml)
[DEBUG] 1002: cmd_execute_and_stream_data() returned 0 after    59016 us
[DEBUG] Trying to hash C:\Dev\vcpkg\scripts\toolchains/windows.cmake
[DEBUG] C:\Dev\vcpkg\scripts\toolchains/windows.cmake has hash e36df1c7f50ab25f9c182fa927d06c19ae082e0d599f132b3f655784b49e4b33
[DEBUG] Trying to hash C:\Dev\vcpkg\triplets\x64-windows.cmake
[DEBUG] C:\Dev\vcpkg\triplets\x64-windows.cmake has hash 4556164a2cd3dd6f4742101eabb46def7e71b6e5856faa88e5d005aac12a803c
Detecting compiler hash for triplet x64-windows...
[DEBUG] Found path: C:\Program Files\Git\cmd\git.exe
[DEBUG] 1003: CreateProcessW("C:\Program Files\Git\cmd\git.exe" --version)
[DEBUG] 1003: cmd_execute_and_stream_data() returned 0 after    68010 us
[DEBUG] Detecting host with IsWow64Process2
[DEBUG] Found path: C:\Program Files\PowerShell\7\pwsh.exe
[DEBUG] 1004: CreateProcessW("C:\Dev\vcpkg-downloads\tools\powershell-core-7.2.11-windows\pwsh.exe" --version)
[DEBUG] 1004: cmd_execute_and_stream_data() returned 0 after   224029 us
[DEBUG] command line: cmd /c "C:\Program Files\Microsoft Visual Studio\2022\Enterprise\VC\Auxiliary/Build\vcvarsall.bat" -vcvars_ver=14.37.32822 amd64   2>&1 <NUL  & echo cdARN4xjKueKScMy9C6H & set
[DEBUG] 1005: CreateProcessW(cmd /c "C:\Program Files\Microsoft Visual Studio\2022\Enterprise\VC\Auxiliary/Build\vcvarsall.bat" -vcvars_ver=14.37.32822 amd64   2>&1 <NUL  & echo cdARN4xjKueKScMy9C6H & set)
[DEBUG] 1005: cmd_execute_and_stream_data() returned 0 after  1401468 us
[DEBUG] **********************************************************************
** Visual Studio 2022 Developer Command Prompt v17.7.3
** Copyright (c) 2022 Microsoft Corporation
**********************************************************************
[vcvarsall.bat] Environment initialized for: 'x64'
cdARN4xjKueKScMy9C6H
ALLUSERSPROFILE=C:\ProgramData
APPDATA=C:\Users\bion\AppData\Roaming
CommandPromptType=Native
CommonProgramFiles=C:\Program Files\Common Files
CommonProgramFiles(x86)=C:\Program Files (x86)\Common Files
CommonProgramW6432=C:\Program Files\Common Files
COMPUTERNAME=BION-TR
ComSpec=C:\Windows\system32\cmd.exe
DevEnvDir=C:\Program Files\Microsoft Visual Studio\2022\Enterprise\Common7\IDE\
ExtensionSdkDir=C:\Program Files (x86)\Microsoft SDKs\Windows Kits\10\ExtensionSDKs
EXTERNAL_INCLUDE=C:\Program Files\Microsoft Visual Studio\2022\Enterprise\VC\Tools\MSVC\14.37.32822\include;C:\Program Files\Microsoft Visual Studio\2022\Enterprise\VC\Tools\MSVC\14.37.32822\ATLMFC\include;C:\Program Files\Microsoft Visual Studio\2022\Enterprise\VC\Auxiliary\VS\include;C:\Program Files (x86)\Windows Kits\10\include\10.0.22621.0\ucrt;C:\Program Files (x86)\Windows Kits\10\\include\10.0.22621.0\\um;C:\Program Files (x86)\Windows Kits\10\\include\10.0.22621.0\\shared;C:\Program Files (x86)\Windows Kits\10\\include\10.0.22621.0\\winrt;C:\Program Files (x86)\Windows Kits\10\\include\10.0.22621.0\\cppwinrt;C:\Program Files (x86)\Windows Kits\NETFXSDK\4.8\include\um
Framework40Version=v4.0
FrameworkDir=C:\Windows\Microsoft.NET\Framework64\
FrameworkDIR64=C:\Windows\Microsoft.NET\Framework64
FrameworkVersion=v4.0.30319
FrameworkVersion64=v4.0.30319
FSHARPINSTALLDIR=C:\Program Files\Microsoft Visual Studio\2022\Enterprise\Common7\IDE\CommonExtensions\Microsoft\FSharp\Tools
HOMEDRIVE=C:
HOMEPATH=\Users\bion
HTMLHelpDir=C:\Program Files (x86)\HTML Help Workshop
INCLUDE=C:\Program Files\Microsoft Visual Studio\2022\Enterprise\VC\Tools\MSVC\14.37.32822\include;C:\Program Files\Microsoft Visual Studio\2022\Enterprise\VC\Tools\MSVC\14.37.32822\ATLMFC\include;C:\Program Files\Microsoft Visual Studio\2022\Enterprise\VC\Auxiliary\VS\include;C:\Program Files (x86)\Windows Kits\10\include\10.0.22621.0\ucrt;C:\Program Files (x86)\Windows Kits\10\\include\10.0.22621.0\\um;C:\Program Files (x86)\Windows Kits\10\\include\10.0.22621.0\\shared;C:\Program Files (x86)\Windows Kits\10\\include\10.0.22621.0\\winrt;C:\Program Files (x86)\Windows Kits\10\\include\10.0.22621.0\\cppwinrt;C:\Program Files (x86)\Windows Kits\NETFXSDK\4.8\include\um
LIB=C:\Program Files\Microsoft Visual Studio\2022\Enterprise\VC\Tools\MSVC\14.37.32822\ATLMFC\lib\x64;C:\Program Files\Microsoft Visual Studio\2022\Enterprise\VC\Tools\MSVC\14.37.32822\lib\x64;C:\Program Files (x86)\Windows Kits\NETFXSDK\4.8\lib\um\x64;C:\Program Files (x86)\Windows Kits\10\lib\10.0.22621.0\ucrt\x64;C:\Program Files (x86)\Windows Kits\10\\lib\10.0.22621.0\\um\x64
LIBPATH=C:\Program Files\Microsoft Visual Studio\2022\Enterprise\VC\Tools\MSVC\14.37.32822\ATLMFC\lib\x64;C:\Program Files\Microsoft Visual Studio\2022\Enterprise\VC\Tools\MSVC\14.37.32822\lib\x64;C:\Program Files\Microsoft Visual Studio\2022\Enterprise\VC\Tools\MSVC\14.37.32822\lib\x86\store\references;C:\Program Files (x86)\Windows Kits\10\UnionMetadata\10.0.22621.0;C:\Program Files (x86)\Windows Kits\10\References\10.0.22621.0;C:\Windows\Microsoft.NET\Framework64\v4.0.30319
LOCALAPPDATA=C:\Users\bion\AppData\Local
LOGONSERVER=\\BION-TR
NETFXSDKDir=C:\Program Files (x86)\Windows Kits\NETFXSDK\4.8\
NUMBER_OF_PROCESSORS=64
OS=Windows_NT
PATHEXT=.COM;.EXE;.BAT;.CMD;.VBS;.VBE;.JS;.JSE;.WSF;.WSH;.MSC;.PY;.PYW;.CPL
Platform=x64
PROCESSOR_ARCHITECTURE=AMD64
PROCESSOR_IDENTIFIER=AMD64 Family 23 Model 49 Stepping 0, AuthenticAMD
PROCESSOR_LEVEL=23
PROCESSOR_REVISION=3100
ProgramData=C:\ProgramData
ProgramFiles=C:\Program Files
ProgramFiles(x86)=C:\Program Files (x86)
ProgramW6432=C:\Program Files
PROMPT=$P$G
PSModulePath=C:\Users\bion\Documents\PowerShell\Modules;C:\Program Files\PowerShell\Modules;c:\program files\powershell\7\Modules;C:\Program Files\WindowsPowerShell\Modules;C:\Windows\system32\WindowsPowerShell\v1.0\Modules
PUBLIC=C:\Users\Public
SystemDrive=C:
SystemRoot=C:\Windows
TEMP=C:\Users\bion\AppData\Local\Temp
TMP=C:\Users\bion\AppData\Local\Temp
UCRTVersion=10.0.22621.0
UniversalCRTSdkDir=C:\Program Files (x86)\Windows Kits\10\
USERDNSDOMAIN=redmond.corp.microsoft.com
USERDOMAIN=REDMOND
USERDOMAIN_ROAMINGPROFILE=REDMOND
USERNAME=bion
USERPROFILE=C:\Users\bion
VCIDEInstallDir=C:\Program Files\Microsoft Visual Studio\2022\Enterprise\Common7\IDE\VC\
VCINSTALLDIR=C:\Program Files\Microsoft Visual Studio\2022\Enterprise\VC\
VCToolsInstallDir=C:\Program Files\Microsoft Visual Studio\2022\Enterprise\VC\Tools\MSVC\14.37.32822\
VCToolsRedistDir=C:\Program Files\Microsoft Visual Studio\2022\Enterprise\VC\Redist\MSVC\14.36.32532\
VCToolsVersion=14.37.32822
VisualStudioVersion=17.0
VS170COMNTOOLS=C:\Program Files\Microsoft Visual Studio\2022\Enterprise\Common7\Tools\
VSCMD_ARG_app_plat=Desktop
VSCMD_ARG_HOST_ARCH=x64
VSCMD_ARG_TGT_ARCH=x64
VSCMD_ARG_VCVARS_VER=14.37.32822
VSCMD_VER=17.7.3
VSINSTALLDIR=C:\Program Files\Microsoft Visual Studio\2022\Enterprise\
windir=C:\Windows
CUDA_PATH=C:\Program Files\NVIDIA GPU Computing Toolkit\CUDA\v12.1
PATH=C:\Program Files\Microsoft Visual Studio\2022\Enterprise\VC\Tools\MSVC\14.37.32822\bin\HostX64\x64;C:\Program Files\Microsoft Visual Studio\2022\Enterprise\Common7\IDE\VC\VCPackages;C:\Program Files\Microsoft Visual Studio\2022\Enterprise\Common7\IDE\CommonExtensions\Microsoft\TestWindow;C:\Program Files\Microsoft Visual Studio\2022\Enterprise\Common7\IDE\CommonExtensions\Microsoft\TeamFoundation\Team Explorer;C:\Program Files\Microsoft Visual Studio\2022\Enterprise\MSBuild\Current\bin\Roslyn;C:\Program Files\Microsoft Visual Studio\2022\Enterprise\Team Tools\Performance Tools\x64;C:\Program Files\Microsoft Visual Studio\2022\Enterprise\Team Tools\Performance Tools;C:\Program Files (x86)\Microsoft Visual Studio\Shared\Common\VSPerfCollectionTools\vs2019\\x64;C:\Program Files (x86)\Microsoft Visual Studio\Shared\Common\VSPerfCollectionTools\vs2019\;C:\Program Files (x86)\Microsoft SDKs\Windows\v10.0A\bin\NETFX 4.8 Tools\x64\;C:\Program Files (x86)\HTML Help Workshop;C:\Program Files\Microsoft Visual Studio\2022\Enterprise\Common7\IDE\CommonExtensions\Microsoft\FSharp\Tools;C:\Program Files\Microsoft Visual Studio\2022\Enterprise\Common7\IDE\Extensions\Microsoft\CodeCoverage.Console;C:\Program Files (x86)\Windows Kits\10\bin\10.0.22621.0\\x64;C:\Program Files (x86)\Windows Kits\10\bin\\x64;C:\Program Files\Microsoft Visual Studio\2022\Enterprise\\MSBuild\Current\Bin\amd64;C:\Windows\Microsoft.NET\Framework64\v4.0.30319;C:\Program Files\Microsoft Visual Studio\2022\Enterprise\Common7\IDE\;C:\Program Files\Microsoft Visual Studio\2022\Enterprise\Common7\Tools\;C:\Dev\vcpkg-downloads\tools\powershell-core-7.2.11-windows;C:\Windows\system32;C:\Windows;C:\Windows\system32\Wbem;C:\Windows\system32\WindowsPowerShell\v1.0\;C:\Program Files\Microsoft Visual Studio\2022\Enterprise\Common7\IDE\CommonExtensions\Microsoft\CMake\CMake\bin;C:\Program Files\Microsoft Visual Studio\2022\Enterprise\Common7\IDE\CommonExtensions\Microsoft\CMake\Ninja;C:\Program Files\Microsoft Visual Studio\2022\Enterprise\Common7\IDE\VC\Linux\bin\ConnectionManagerExe
VSLANG=1033
VSCMD_SKIP_SENDTELEMETRY=1
VCPKG_COMMAND=C:/Dev/vcpkg-tool/out/build/Win-x64-Debug-WithArtifacts/vcpkg.exe
WindowsLibPath=C:\Program Files (x86)\Windows Kits\10\UnionMetadata\10.0.22621.0;C:\Program Files (x86)\Windows Kits\10\References\10.0.22621.0
WindowsSdkBinPath=C:\Program Files (x86)\Windows Kits\10\bin\
WindowsSdkDir=C:\Program Files (x86)\Windows Kits\10\
WindowsSDKLibVersion=10.0.22621.0\
WindowsSdkVerBinPath=C:\Program Files (x86)\Windows Kits\10\bin\10.0.22621.0\
WindowsSDKVersion=10.0.22621.0\
WindowsSDK_ExecutablePath_x64=C:\Program Files (x86)\Microsoft SDKs\Windows\v10.0A\bin\NETFX 4.8 Tools\x64\
WindowsSDK_ExecutablePath_x86=C:\Program Files (x86)\Microsoft SDKs\Windows\v10.0A\bin\NETFX 4.8 Tools\
X_VCPKG_RECURSIVE_DATA={
"VCPKG_ROOT_ENV": "C:\\Dev\\vcpkg",
"VCPKG_DOWNLOADS": "C:\\Dev\\vcpkg-downloads"
}

__DOTNET_ADD_64BIT=1
__DOTNET_PREFERRED_BITNESS=64
__VSCMD_PREINIT_PATH=C:\Dev\vcpkg-downloads\tools\powershell-core-7.2.11-windows;C:\Windows\system32;C:\Windows;C:\Windows\system32\Wbem;C:\Windows\system32\WindowsPowerShell\v1.0\

[DEBUG] 1006: CreateProcessW("C:\Dev\vcpkg-downloads\tools\cmake-3.27.1-windows\cmake-3.27.1-windows-i386/bin/cmake.exe" -DCURRENT_PORT_DIR=C:/Dev/vcpkg/scripts/detect_compiler -DCURRENT_BUILDTREES_DIR=C:/Dev/vcpkg/buildtrees/detect_compiler -DCURRENT_PACKAGES_DIR=C:/Dev/vcpkg/packages/detect_compiler_x64-windows -D_HOST_TRIPLET=x64-windows -DCMD=BUILD -DDOWNLOADS=C:/Dev/vcpkg-downloads -DTARGET_TRIPLET=x64-windows -DTARGET_TRIPLET_FILE=C:/Dev/vcpkg/triplets/x64-windows.cmake -DVCPKG_BASE_VERSION=2999-12-31 -DVCPKG_CONCURRENCY=65 -DVCPKG_PLATFORM_TOOLSET=v143 "-DGIT=C:/Program Files/Git/cmd/git.exe" -DVCPKG_ROOT_DIR=C:/Dev/vcpkg -DPACKAGES_DIR=C:/Dev/vcpkg/packages -DBUILDTREES_DIR=C:/Dev/vcpkg/buildtrees -D_VCPKG_INSTALLED_DIR=C:/Dev/vcpkg/installed -DDOWNLOADS=C:/Dev/vcpkg-downloads -DVCPKG_MANIFEST_INSTALL=OFF -P "C:\Dev\vcpkg\scripts\ports.cmake")
[DEBUG] -- Configuring x64-windows
[DEBUG] [1/1] "C:/Dev/vcpkg-downloads/tools/cmake-3.27.1-windows/cmake-3.27.1-windows-i386/bin/cmake.exe" -E chdir ".." "C:/Dev/vcpkg-downloads/tools/cmake-3.27.1-windows/cmake-3.27.1-windows-i386/bin/cmake.exe" "C:/Dev/vcpkg/scripts/detect_compiler" "-G" "Ninja" "-DCMAKE_BUILD_TYPE=Release" "-DCMAKE_INSTALL_PREFIX=C:/Dev/vcpkg/packages/detect_compiler_x64-windows" "-DCMAKE_MAKE_PROGRAM=C:/Dev/vcpkg-downloads/tools/ninja/1.10.2-windows/ninja.exe" "-DBUILD_SHARED_LIBS=ON" "-DVCPKG_CHAINLOAD_TOOLCHAIN_FILE=C:/Dev/vcpkg/scripts/toolchains/windows.cmake" "-DVCPKG_TARGET_TRIPLET=x64-windows" "-DVCPKG_SET_CHARSET_FLAG=ON" "-DVCPKG_PLATFORM_TOOLSET=v143" "-DCMAKE_EXPORT_NO_PACKAGE_REGISTRY=ON" "-DCMAKE_FIND_PACKAGE_NO_PACKAGE_REGISTRY=ON" "-DCMAKE_FIND_PACKAGE_NO_SYSTEM_PACKAGE_REGISTRY=ON" "-DCMAKE_INSTALL_SYSTEM_RUNTIME_LIBS_SKIP=TRUE" "-DCMAKE_VERBOSE_MAKEFILE=ON" "-DVCPKG_APPLOCAL_DEPS=OFF" "-DCMAKE_TOOLCHAIN_FILE=C:/Dev/vcpkg/scripts/buildsystems/vcpkg.cmake" "-DCMAKE_ERROR_ON_ABSOLUTE_INSTALL_DESTINATION=ON" "-DVCPKG_CXX_FLAGS=" "-DVCPKG_CXX_FLAGS_RELEASE=" "-DVCPKG_CXX_FLAGS_DEBUG=" "-DVCPKG_C_FLAGS=" "-DVCPKG_C_FLAGS_RELEASE=" "-DVCPKG_C_FLAGS_DEBUG=" "-DVCPKG_CRT_LINKAGE=dynamic" "-DVCPKG_LINKER_FLAGS=" "-DVCPKG_LINKER_FLAGS_RELEASE=" "-DVCPKG_LINKER_FLAGS_DEBUG=" "-DVCPKG_TARGET_ARCHITECTURE=x64" "-DCMAKE_INSTALL_LIBDIR:STRING=lib" "-DCMAKE_INSTALL_BINDIR:STRING=bin" "-D_VCPKG_ROOT_DIR=C:/Dev/vcpkg" "-DZ_VCPKG_ROOT_DIR=C:/Dev/vcpkg" "-D_VCPKG_INSTALLED_DIR=C:/Dev/vcpkg/installed" "-DVCPKG_MANIFEST_INSTALL=OFF"
[DEBUG] -- The C compiler identification is MSVC 19.37.32822.0
[DEBUG] -- The CXX compiler identification is MSVC 19.37.32822.0
[DEBUG] #COMPILER_HASH#f4a88a8e9062ec73eb4a671e8bb00c783a16b73c
[DEBUG] #COMPILER_C_HASH#1905024596298b064b313c23781fe351623dc957
[DEBUG] #COMPILER_C_VERSION#19.37.32822.0
[DEBUG] #COMPILER_C_ID#MSVC
[DEBUG] #COMPILER_CXX_HASH#1905024596298b064b313c23781fe351623dc957
[DEBUG] #COMPILER_CXX_VERSION#19.37.32822.0
[DEBUG] #COMPILER_CXX_ID#MSVC
[DEBUG] -- Configuring done (1.3s)
[DEBUG] -- Generating done (0.0s)
[DEBUG] CMake Warning:
[DEBUG]   Manually-specified variables were not used by the project:
[DEBUG]
[DEBUG]     BUILD_SHARED_LIBS
[DEBUG]     CMAKE_INSTALL_BINDIR
[DEBUG]     CMAKE_INSTALL_LIBDIR
[DEBUG]     _VCPKG_ROOT_DIR
[DEBUG]
[DEBUG]
[DEBUG] -- Build files have been written to: C:/Dev/vcpkg/buildtrees/detect_compiler/x64-windows-rel
[DEBUG]
[DEBUG] 1006: cmd_execute_and_stream_data() returned 0 after  1624848 us
[DEBUG] Detected compiler hash for triplet x64-windows: f4a88a8e9062ec73eb4a671e8bb00c783a16b73c
[DEBUG] Trying to hash C:\Dev\vcpkg\ports\zlib\0001-Prevent-invalid-inclusions-when-HAVE_-is-set-to-0.patch
[DEBUG] C:\Dev\vcpkg\ports\zlib\0001-Prevent-invalid-inclusions-when-HAVE_-is-set-to-0.patch has hash 750b9542cb55e6328cca01d3ca997f1373b9530afa95e04213168676936e7bfa
[DEBUG] Trying to hash C:\Dev\vcpkg\ports\zlib\0002-skip-building-examples.patch
[DEBUG] C:\Dev\vcpkg\ports\zlib\0002-skip-building-examples.patch has hash 835ddecfed752e0f49be9b0f8ff7ba76541cb0a150044327316e22ca84f8d0c2
[DEBUG] Trying to hash C:\Dev\vcpkg\ports\zlib\0003-build-static-or-shared-not-both.patch
[DEBUG] C:\Dev\vcpkg\ports\zlib\0003-build-static-or-shared-not-both.patch has hash d6026271dcb3d8fc74b41e235620ae31576a798e77aa411c3af8cd9e948c02b1
[DEBUG] Trying to hash C:\Dev\vcpkg\ports\zlib\0004-android-and-mingw-fixes.patch
[DEBUG] C:\Dev\vcpkg\ports\zlib\0004-android-and-mingw-fixes.patch has hash 37a43eddbcb1b7dde49e7659ae895dfd0ff1df66666c1371ba7d5bfc49d8b438
[DEBUG] Trying to hash C:\Dev\vcpkg\ports\zlib\usage
[DEBUG] C:\Dev\vcpkg\ports\zlib\usage has hash be22662327df993eebc437495add75acb365ab18d37c7e5de735d4ea4f5d3083
[DEBUG] Trying to hash C:\Dev\vcpkg\ports\zlib\vcpkg.json
[DEBUG] C:\Dev\vcpkg\ports\zlib\vcpkg.json has hash bc94e2540efabe36130a806381a001c57194e7de67454ab7ff1e30aa15e6ce23
[DEBUG] Trying to hash C:\Dev\vcpkg\scripts\cmake\compile_wrapper_consider_clang-cl.patch
[DEBUG] C:\Dev\vcpkg\scripts\cmake\compile_wrapper_consider_clang-cl.patch has hash 12dbcf92febc9400d782a3d7d073acee4b09a1417ce7d4d5e9be7c55aa177d36
[DEBUG] Trying to hash C:\Dev\vcpkg\scripts\cmake\execute_process.cmake
[DEBUG] C:\Dev\vcpkg\scripts\cmake\execute_process.cmake has hash 66a937b9c074422643135c319d1abadaa45484a664f1b160d4c163efb444a446
[DEBUG] Trying to hash C:\Dev\vcpkg\scripts\cmake\vcpkg_acquire_msys.cmake
[DEBUG] C:\Dev\vcpkg\scripts\cmake\vcpkg_acquire_msys.cmake has hash 439bdfd5ffb301ba19a974e7c759dacea6750442ec207f2d0520a1faf11301c0
[DEBUG] Trying to hash C:\Dev\vcpkg\scripts\cmake\vcpkg_add_to_path.cmake
[DEBUG] C:\Dev\vcpkg\scripts\cmake\vcpkg_add_to_path.cmake has hash 5f5ae75cf37b2a58d1a8561ca96496b64cd91ec9a0afab0b976c3e5d59030bfe
[DEBUG] Trying to hash C:\Dev\vcpkg\scripts\cmake\vcpkg_apply_patches.cmake
[DEBUG] C:\Dev\vcpkg\scripts\cmake\vcpkg_apply_patches.cmake has hash 4c123ce6f9bf0c46adc8287b7bd6a40b5f5e8daa83623dd21bb8c568ad4421fc
[DEBUG] Trying to hash C:\Dev\vcpkg\scripts\cmake\vcpkg_backup_restore_env_vars.cmake
[DEBUG] C:\Dev\vcpkg\scripts\cmake\vcpkg_backup_restore_env_vars.cmake has hash 5621f3b6a9586ffc02d49a5032560a48ad291de1cce553b904588f0108648904
[DEBUG] Trying to hash C:\Dev\vcpkg\scripts\cmake\vcpkg_buildpath_length_warning.cmake
[DEBUG] C:\Dev\vcpkg\scripts\cmake\vcpkg_buildpath_length_warning.cmake has hash fa440f3734e34b462ef96f3e89f0a9c5a11b609110a15a162d24da9218693fd1
[DEBUG] Trying to hash C:\Dev\vcpkg\scripts\cmake\vcpkg_build_cmake.cmake
[DEBUG] C:\Dev\vcpkg\scripts\cmake\vcpkg_build_cmake.cmake has hash 5201d6149d85fff414f3fbbdc4c793e0b2aeeb6d249a5ae8cf15e39fef66b7e6
[DEBUG] Trying to hash C:\Dev\vcpkg\scripts\cmake\vcpkg_build_make.cmake
[DEBUG] C:\Dev\vcpkg\scripts\cmake\vcpkg_build_make.cmake has hash 665ce441da618e4270950034db2912f6a863689c260a8a889d4ad41ff84367ba
[DEBUG] Trying to hash C:\Dev\vcpkg\scripts\cmake\vcpkg_build_msbuild.cmake
[DEBUG] C:\Dev\vcpkg\scripts\cmake\vcpkg_build_msbuild.cmake has hash 9a8367ac19733fc51b8e7edc77968725d2efe1400aaf1fdd767bb329a7b04532
[DEBUG] Trying to hash C:\Dev\vcpkg\scripts\cmake\vcpkg_build_ninja.cmake
[DEBUG] C:\Dev\vcpkg\scripts\cmake\vcpkg_build_ninja.cmake has hash 9bafdd942fe41be20a868e0a41ea6f983895bacba2f257921b53d789c7ad8092
[DEBUG] Trying to hash C:\Dev\vcpkg\scripts\cmake\vcpkg_build_nmake.cmake
[DEBUG] C:\Dev\vcpkg\scripts\cmake\vcpkg_build_nmake.cmake has hash 918422e54a328b9dce1be8c06916d612de9cc24a18c65cc47f26d78e8c487440
[DEBUG] Trying to hash C:\Dev\vcpkg\scripts\cmake\vcpkg_build_qmake.cmake
[DEBUG] C:\Dev\vcpkg\scripts\cmake\vcpkg_build_qmake.cmake has hash cd00146eb665b004c527cf74986e392d2419610ace073bd6abb79d55a7912004
[DEBUG] Trying to hash C:\Dev\vcpkg\scripts\cmake\vcpkg_check_features.cmake
[DEBUG] C:\Dev\vcpkg\scripts\cmake\vcpkg_check_features.cmake has hash 943b217e0968d64cf2cb9c272608e6a0b497377e792034f819809a79e1502c2b
[DEBUG] Trying to hash C:\Dev\vcpkg\scripts\cmake\vcpkg_check_linkage.cmake
[DEBUG] C:\Dev\vcpkg\scripts\cmake\vcpkg_check_linkage.cmake has hash 2236edfea13af20524c17d73214e0e5111bf1c13a2fc46c7f2912a46abd0c1a8
[DEBUG] Trying to hash C:\Dev\vcpkg\scripts\cmake\vcpkg_clean_executables_in_bin.cmake
[DEBUG] C:\Dev\vcpkg\scripts\cmake\vcpkg_clean_executables_in_bin.cmake has hash 5f7427ddf120583cf1f429861989e937858ee7a8c4deb419dca6f3f2fee86a65
[DEBUG] Trying to hash C:\Dev\vcpkg\scripts\cmake\vcpkg_clean_msbuild.cmake
[DEBUG] C:\Dev\vcpkg\scripts\cmake\vcpkg_clean_msbuild.cmake has hash 67007e54dee8a38487b43228573d877ad270fc6bac7bf8849783d0c2a78ce501
[DEBUG] Trying to hash C:\Dev\vcpkg\scripts\cmake\vcpkg_common_definitions.cmake
[DEBUG] C:\Dev\vcpkg\scripts\cmake\vcpkg_common_definitions.cmake has hash ea01c5711c7da32d9037361edee123507cb3708fe43b40ab3ecbbd32872310e3
[DEBUG] Trying to hash C:\Dev\vcpkg\scripts\cmake\vcpkg_common_functions.cmake
[DEBUG] C:\Dev\vcpkg\scripts\cmake\vcpkg_common_functions.cmake has hash 5f07e6f6184a02dcd8fc058eed1747a6bb65ee4a8f524e717651674f0bd7904c
[DEBUG] Trying to hash C:\Dev\vcpkg\scripts\cmake\vcpkg_configure_cmake.cmake
[DEBUG] C:\Dev\vcpkg\scripts\cmake\vcpkg_configure_cmake.cmake has hash 9dfd362bd20613eaa83af55eb0f98c8cb50fd4826a65da74a0f1641da73497c2
[DEBUG] Trying to hash C:\Dev\vcpkg\scripts\cmake\vcpkg_configure_gn.cmake
[DEBUG] C:\Dev\vcpkg\scripts\cmake\vcpkg_configure_gn.cmake has hash c6637dbdb7c20b2e067c924eeb50605f7e3cc9e7c8c1434e53c9280134df11ba
[DEBUG] Trying to hash C:\Dev\vcpkg\scripts\cmake\vcpkg_configure_make.cmake
[DEBUG] C:\Dev\vcpkg\scripts\cmake\vcpkg_configure_make.cmake has hash d406467644b16f330eae5acb1bc91e5118b3d547d3a72547eb7db864ef62af84
[DEBUG] Trying to hash C:\Dev\vcpkg\scripts\cmake\vcpkg_configure_meson.cmake
[DEBUG] C:\Dev\vcpkg\scripts\cmake\vcpkg_configure_meson.cmake has hash 23c481b5e1bb9d7a4877608fd327d7289601331c8a10259922a90aec5addf8a2
[DEBUG] Trying to hash C:\Dev\vcpkg\scripts\cmake\vcpkg_configure_qmake.cmake
[DEBUG] C:\Dev\vcpkg\scripts\cmake\vcpkg_configure_qmake.cmake has hash 54d2123ecd337643ce9f3b0e4ef48f092bee5aea84d0372d12d1cc5f209a6437
[DEBUG] Trying to hash C:\Dev\vcpkg\scripts\cmake\vcpkg_copy_pdbs.cmake
[DEBUG] C:\Dev\vcpkg\scripts\cmake\vcpkg_copy_pdbs.cmake has hash d57e4f196c82dc562a9968c6155073094513c31e2de475694143d3aa47954b1c
[DEBUG] Trying to hash C:\Dev\vcpkg\scripts\cmake\vcpkg_copy_tools.cmake
[DEBUG] C:\Dev\vcpkg\scripts\cmake\vcpkg_copy_tools.cmake has hash 3d45ff761bddbabe8923b52330168dc3abd295fa469d3f2e47cb14dce85332d5
[DEBUG] Trying to hash C:\Dev\vcpkg\scripts\cmake\vcpkg_copy_tool_dependencies.cmake
[DEBUG] C:\Dev\vcpkg\scripts\cmake\vcpkg_copy_tool_dependencies.cmake has hash 66eef4116a52fd8cba76d274f44904846df943a3fc52317871cf255733b2efc4
[DEBUG] Trying to hash C:\Dev\vcpkg\scripts\cmake\vcpkg_download_distfile.cmake
[DEBUG] C:\Dev\vcpkg\scripts\cmake\vcpkg_download_distfile.cmake has hash 2d57e1336ae9daab2f04c0623217094b026080d19bdca3a92396d68ea3e729c7
[DEBUG] Trying to hash C:\Dev\vcpkg\scripts\cmake\vcpkg_execute_build_process.cmake
[DEBUG] C:\Dev\vcpkg\scripts\cmake\vcpkg_execute_build_process.cmake has hash 4145b3b23fcc2dd94a0fe3c818b63fcea40854e5aa94d08dc83ddcc09727cedf
[DEBUG] Trying to hash C:\Dev\vcpkg\scripts\cmake\vcpkg_execute_in_download_mode.cmake
[DEBUG] C:\Dev\vcpkg\scripts\cmake\vcpkg_execute_in_download_mode.cmake has hash 75a3cabe04d6bbaf1f73555516dd3c492edbed22cea204835c3620db5d948851
[DEBUG] Trying to hash C:\Dev\vcpkg\scripts\cmake\vcpkg_execute_required_process.cmake
[DEBUG] C:\Dev\vcpkg\scripts\cmake\vcpkg_execute_required_process.cmake has hash 0c6f52ca848715efd88acfec9bdb2ed3b5f38a650baa6a614bf23ccc3eec150a
[DEBUG] Trying to hash C:\Dev\vcpkg\scripts\cmake\vcpkg_execute_required_process_repeat.cmake
[DEBUG] C:\Dev\vcpkg\scripts\cmake\vcpkg_execute_required_process_repeat.cmake has hash 4863f71de8d0fd53ed799c6a5d15795156ae0606ecb61489b7636a050e029e2b
[DEBUG] Trying to hash C:\Dev\vcpkg\scripts\cmake\vcpkg_extract_source_archive.cmake
[DEBUG] C:\Dev\vcpkg\scripts\cmake\vcpkg_extract_source_archive.cmake has hash a25e0132b36c30e9675f934042fb1164128864eeefa064ab8eabb1eb347aa381
[DEBUG] Trying to hash C:\Dev\vcpkg\scripts\cmake\vcpkg_extract_source_archive_ex.cmake
[DEBUG] C:\Dev\vcpkg\scripts\cmake\vcpkg_extract_source_archive_ex.cmake has hash ab9f5bd71a8e1ca4a56d5c09f248c2bc10ff8df3b600c3efb6cca34cba9ec3b7
[DEBUG] Trying to hash C:\Dev\vcpkg\scripts\cmake\vcpkg_fail_port_install.cmake
[DEBUG] C:\Dev\vcpkg\scripts\cmake\vcpkg_fail_port_install.cmake has hash aef80df96f7c83d79e7b03d132161d8071228e0f1c9eb61ba8b16354f0c9496c
[DEBUG] Trying to hash C:\Dev\vcpkg\scripts\cmake\vcpkg_find_acquire_program(7Z).cmake
[DEBUG] C:\Dev\vcpkg\scripts\cmake\vcpkg_find_acquire_program(7Z).cmake has hash d90166548324314112ecaebc73ae440419ff2393c4882ff6420ebacb57923ea0
[DEBUG] Trying to hash C:\Dev\vcpkg\scripts\cmake\vcpkg_find_acquire_program(ARIA2).cmake
[DEBUG] C:\Dev\vcpkg\scripts\cmake\vcpkg_find_acquire_program(ARIA2).cmake has hash 0d4fdc0e24f6bfff5246bbc92c29c1a3bf8996452c5f85c26b667735d78b1a0b
[DEBUG] Trying to hash C:\Dev\vcpkg\scripts\cmake\vcpkg_find_acquire_program(BAZEL).cmake
[DEBUG] C:\Dev\vcpkg\scripts\cmake\vcpkg_find_acquire_program(BAZEL).cmake has hash 4a22f3ef45670639e38af6040a3fb2c99b7c719082e2ce87e082c9f8ee91b840
[DEBUG] Trying to hash C:\Dev\vcpkg\scripts\cmake\vcpkg_find_acquire_program(BISON).cmake
[DEBUG] C:\Dev\vcpkg\scripts\cmake\vcpkg_find_acquire_program(BISON).cmake has hash 4df1053ab129f7b9ccd5d58baaa91628c9269e259a7239c7eaed795ff1cae1d1
[DEBUG] Trying to hash C:\Dev\vcpkg\scripts\cmake\vcpkg_find_acquire_program(CLANG).cmake
[DEBUG] C:\Dev\vcpkg\scripts\cmake\vcpkg_find_acquire_program(CLANG).cmake has hash bc9f8433d5ce5a51e45a79c7469cdd9fcad154488b33ff6353af3b29400af50d
[DEBUG] Trying to hash C:\Dev\vcpkg\scripts\cmake\vcpkg_find_acquire_program(DARK).cmake
[DEBUG] C:\Dev\vcpkg\scripts\cmake\vcpkg_find_acquire_program(DARK).cmake has hash f20fed20d3fbf637ac329e1aae27193485d9a2954ae5fb21b5f00272b387286d
[DEBUG] Trying to hash C:\Dev\vcpkg\scripts\cmake\vcpkg_find_acquire_program(DOXYGEN).cmake
[DEBUG] C:\Dev\vcpkg\scripts\cmake\vcpkg_find_acquire_program(DOXYGEN).cmake has hash e3dcfd0b6a64f77db57a00edd22171f426103a6235165c906a38826e75c041b7
[DEBUG] Trying to hash C:\Dev\vcpkg\scripts\cmake\vcpkg_find_acquire_program(FLEX).cmake
[DEBUG] C:\Dev\vcpkg\scripts\cmake\vcpkg_find_acquire_program(FLEX).cmake has hash 5126699579bda4fb2b605adedd8cfed7ac53430178931ffd20d8b007cc89f26f
[DEBUG] Trying to hash C:\Dev\vcpkg\scripts\cmake\vcpkg_find_acquire_program(GASPREPROCESSOR).cmake
[DEBUG] C:\Dev\vcpkg\scripts\cmake\vcpkg_find_acquire_program(GASPREPROCESSOR).cmake has hash 0ff31ab283d7b78bc7af20cdbd66b5adc0432f00f126f8111003245908398c88
[DEBUG] Trying to hash C:\Dev\vcpkg\scripts\cmake\vcpkg_find_acquire_program(GIT).cmake
[DEBUG] C:\Dev\vcpkg\scripts\cmake\vcpkg_find_acquire_program(GIT).cmake has hash bb3644b8863c1a03c3c76cf8b13047557e05b22f4be7a13cb4ad00e72d7ad6b6
[DEBUG] Trying to hash C:\Dev\vcpkg\scripts\cmake\vcpkg_find_acquire_program(GN).cmake
[DEBUG] C:\Dev\vcpkg\scripts\cmake\vcpkg_find_acquire_program(GN).cmake has hash f6c65fce5a51b65b8d081b05f11ae186223b39d78d2eeaf836f66b4e2e515aa3
[DEBUG] Trying to hash C:\Dev\vcpkg\scripts\cmake\vcpkg_find_acquire_program(GO).cmake
[DEBUG] C:\Dev\vcpkg\scripts\cmake\vcpkg_find_acquire_program(GO).cmake has hash 9fd39e73f7e9bdf425a808b250fbdd07f42e9424319b9d7a66b9d5a08a6f0b51
[DEBUG] Trying to hash C:\Dev\vcpkg\scripts\cmake\vcpkg_find_acquire_program(GPERF).cmake
[DEBUG] C:\Dev\vcpkg\scripts\cmake\vcpkg_find_acquire_program(GPERF).cmake has hash 51821386b816ca387803930f656ef5bab21ff940bd78481ff7d5be2b1dad08a6
[DEBUG] Trying to hash C:\Dev\vcpkg\scripts\cmake\vcpkg_find_acquire_program(JOM).cmake
[DEBUG] C:\Dev\vcpkg\scripts\cmake\vcpkg_find_acquire_program(JOM).cmake has hash 0bd0d635ecc1a9532d2c95f4f7d4a6ec5a69d210309745e02a549841770a7773
[DEBUG] Trying to hash C:\Dev\vcpkg\scripts\cmake\vcpkg_find_acquire_program(MESON).cmake
[DEBUG] C:\Dev\vcpkg\scripts\cmake\vcpkg_find_acquire_program(MESON).cmake has hash a8c8e3b3cb948ca4d7c8b0b6ecce4006f82a361ec60291afe597c2a7e24300b0
[DEBUG] Trying to hash C:\Dev\vcpkg\scripts\cmake\vcpkg_find_acquire_program(NASM).cmake
[DEBUG] C:\Dev\vcpkg\scripts\cmake\vcpkg_find_acquire_program(NASM).cmake has hash 4682f861100163f7d3290dd7c64c86dd63f1447c2726d4449a46bc821d8a0a21
[DEBUG] Trying to hash C:\Dev\vcpkg\scripts\cmake\vcpkg_find_acquire_program(NINJA).cmake
[DEBUG] C:\Dev\vcpkg\scripts\cmake\vcpkg_find_acquire_program(NINJA).cmake has hash c0d488f37ddbef505e2e3ffaf588bc1da18f64da68089c9d325affea1a62f1a6
[DEBUG] Trying to hash C:\Dev\vcpkg\scripts\cmake\vcpkg_find_acquire_program(NUGET).cmake
[DEBUG] C:\Dev\vcpkg\scripts\cmake\vcpkg_find_acquire_program(NUGET).cmake has hash b086c0d991f8aab5351180ab2922410d061b7765eea9d64245fbacfff16a96c9
[DEBUG] Trying to hash C:\Dev\vcpkg\scripts\cmake\vcpkg_find_acquire_program(PATCHELF).cmake
[DEBUG] C:\Dev\vcpkg\scripts\cmake\vcpkg_find_acquire_program(PATCHELF).cmake has hash 11f2f903a5c7334e72ddae2cc18ad9d14040731961a0455ef2cf63aa552bd6e8
[DEBUG] Trying to hash C:\Dev\vcpkg\scripts\cmake\vcpkg_find_acquire_program(PERL).cmake
[DEBUG] C:\Dev\vcpkg\scripts\cmake\vcpkg_find_acquire_program(PERL).cmake has hash e839596a2b9dba81d28d2df0f7ed5fe7e766f45bb8ab6130634a7f7269db1f09
[DEBUG] Trying to hash C:\Dev\vcpkg\scripts\cmake\vcpkg_find_acquire_program(PKGCONFIG).cmake
[DEBUG] C:\Dev\vcpkg\scripts\cmake\vcpkg_find_acquire_program(PKGCONFIG).cmake has hash e6e05b67513c71a95cf0dcd6f9f8a014c97d90ba82a874c8ce31a2b5ffb952b0
[DEBUG] Trying to hash C:\Dev\vcpkg\scripts\cmake\vcpkg_find_acquire_program(PYTHON2).cmake
[DEBUG] C:\Dev\vcpkg\scripts\cmake\vcpkg_find_acquire_program(PYTHON2).cmake has hash ed078b84af22461e59694ab205a5ba9d0eb77d094ea64fc5754f7e19644b052e
[DEBUG] Trying to hash C:\Dev\vcpkg\scripts\cmake\vcpkg_find_acquire_program(PYTHON3).cmake
[DEBUG] C:\Dev\vcpkg\scripts\cmake\vcpkg_find_acquire_program(PYTHON3).cmake has hash bf78099b5a097427eb6fbaa1dabe3eccea50abbcfe37bfc89e6c9ad8c6c9cb90
[DEBUG] Trying to hash C:\Dev\vcpkg\scripts\cmake\vcpkg_find_acquire_program(RUBY).cmake
[DEBUG] C:\Dev\vcpkg\scripts\cmake\vcpkg_find_acquire_program(RUBY).cmake has hash d8f4817a1586c627d680bba1eea42194351746bacc02f35f2eedbb3cd8e021b2
[DEBUG] Trying to hash C:\Dev\vcpkg\scripts\cmake\vcpkg_find_acquire_program(SCONS).cmake
[DEBUG] C:\Dev\vcpkg\scripts\cmake\vcpkg_find_acquire_program(SCONS).cmake has hash a9e54f49772920491ef5ed5c4633869d917f92ba7a9b222771124eab16ea006b
[DEBUG] Trying to hash C:\Dev\vcpkg\scripts\cmake\vcpkg_find_acquire_program(SWIG).cmake
[DEBUG] C:\Dev\vcpkg\scripts\cmake\vcpkg_find_acquire_program(SWIG).cmake has hash 69cbee18ee009cd0b9da6ea9a028a66761abaa8eda3186ccc258b729122d6435
[DEBUG] Trying to hash C:\Dev\vcpkg\scripts\cmake\vcpkg_find_acquire_program(YASM).cmake
[DEBUG] C:\Dev\vcpkg\scripts\cmake\vcpkg_find_acquire_program(YASM).cmake has hash 452a03ef1963d7773b515dde97f783cf7db5b5ecb7ad2df7bf33c7612322820d
[DEBUG] Trying to hash C:\Dev\vcpkg\scripts\cmake\vcpkg_find_acquire_program.cmake
[DEBUG] C:\Dev\vcpkg\scripts\cmake\vcpkg_find_acquire_program.cmake has hash 722996708543bc3e10f7ae54acc4a6468478c28adb52c84a930d096ff56f76dd
[DEBUG] Trying to hash C:\Dev\vcpkg\scripts\cmake\vcpkg_find_fortran.cmake
[DEBUG] C:\Dev\vcpkg\scripts\cmake\vcpkg_find_fortran.cmake has hash 0a5d03ecbe6ebd8cc9a654bc8e0441f35167ec5452d44791fe9b1c453dbb2ed9
[DEBUG] Trying to hash C:\Dev\vcpkg\scripts\cmake\vcpkg_fixup_cmake_targets.cmake
[DEBUG] C:\Dev\vcpkg\scripts\cmake\vcpkg_fixup_cmake_targets.cmake has hash 7f777f83ca76540ed573d986ee53d1b3da1e45f49f3856c44c0773fec71cb49f
[DEBUG] Trying to hash C:\Dev\vcpkg\scripts\cmake\vcpkg_fixup_pkgconfig.cmake
[DEBUG] C:\Dev\vcpkg\scripts\cmake\vcpkg_fixup_pkgconfig.cmake has hash 588d833ff057d3ca99c14616c7ecfb5948b5e2a9e4fc02517dceb8b803473457
[DEBUG] Trying to hash C:\Dev\vcpkg\scripts\cmake\vcpkg_from_bitbucket.cmake
[DEBUG] C:\Dev\vcpkg\scripts\cmake\vcpkg_from_bitbucket.cmake has hash e875eb81fb8b06b08d0c97aed211b161dca3d1b7da8e562aa5ee8a8809e9e3d0
[DEBUG] Trying to hash C:\Dev\vcpkg\scripts\cmake\vcpkg_from_git.cmake
[DEBUG] C:\Dev\vcpkg\scripts\cmake\vcpkg_from_git.cmake has hash 8f27bff0d01c6d15a3e691758df52bfbb0b1b929da45c4ebba02ef76b54b1881
[DEBUG] Trying to hash C:\Dev\vcpkg\scripts\cmake\vcpkg_from_github.cmake
[DEBUG] C:\Dev\vcpkg\scripts\cmake\vcpkg_from_github.cmake has hash b743742296a114ea1b18ae99672e02f142c4eb2bef7f57d36c038bedbfb0502f
[DEBUG] Trying to hash C:\Dev\vcpkg\scripts\cmake\vcpkg_from_gitlab.cmake
[DEBUG] C:\Dev\vcpkg\scripts\cmake\vcpkg_from_gitlab.cmake has hash 656b2e2f210c6607908ae03771b049648b55213ef0a782135f90e44edb271fc5
[DEBUG] Trying to hash C:\Dev\vcpkg\scripts\cmake\vcpkg_from_sourceforge.cmake
[DEBUG] C:\Dev\vcpkg\scripts\cmake\vcpkg_from_sourceforge.cmake has hash 99f72faf1e7611ac9c367677457e29599d81bb86f204ec68a3f0df87a62f0922
[DEBUG] Trying to hash C:\Dev\vcpkg\scripts\cmake\vcpkg_get_program_files_platform_bitness.cmake
[DEBUG] C:\Dev\vcpkg\scripts\cmake\vcpkg_get_program_files_platform_bitness.cmake has hash 0c21ce7e9fcf7828f2e06f488f4675816b87942cfbf828c183f5bbcba8ae5fc7
[DEBUG] Trying to hash C:\Dev\vcpkg\scripts\cmake\vcpkg_get_windows_sdk.cmake
[DEBUG] C:\Dev\vcpkg\scripts\cmake\vcpkg_get_windows_sdk.cmake has hash 9fe97b3807d78c81b54b9512e920112baf68c2b4f5ce483b2764edada074b4ec
[DEBUG] Trying to hash C:\Dev\vcpkg\scripts\cmake\vcpkg_host_path_list.cmake
[DEBUG] C:\Dev\vcpkg\scripts\cmake\vcpkg_host_path_list.cmake has hash 1e8382b3a80fc7458293fce1659077dcacda9892eb63d911ae0330fdb1aedc2d
[DEBUG] Trying to hash C:\Dev\vcpkg\scripts\cmake\vcpkg_install_cmake.cmake
[DEBUG] C:\Dev\vcpkg\scripts\cmake\vcpkg_install_cmake.cmake has hash 6119891902c6ad9e9ed1d9b073eb9969159b553f9d6f5bee529cde91769355ff
[DEBUG] Trying to hash C:\Dev\vcpkg\scripts\cmake\vcpkg_install_copyright.cmake
[DEBUG] C:\Dev\vcpkg\scripts\cmake\vcpkg_install_copyright.cmake has hash ba6c169ab4e59fa05682e530cdeb883767de22c8391f023d4e6844a7ec5dd3d2
[DEBUG] Trying to hash C:\Dev\vcpkg\scripts\cmake\vcpkg_install_gn.cmake
[DEBUG] C:\Dev\vcpkg\scripts\cmake\vcpkg_install_gn.cmake has hash 18f1e80697fca670452e1ff181850e546764cc7c5fa8cb5d6442266523af9806
[DEBUG] Trying to hash C:\Dev\vcpkg\scripts\cmake\vcpkg_install_make.cmake
[DEBUG] C:\Dev\vcpkg\scripts\cmake\vcpkg_install_make.cmake has hash 8d8145837ce718ef60778b50fea1af272146c9efa42367ec2d7726332ef29ab5
[DEBUG] Trying to hash C:\Dev\vcpkg\scripts\cmake\vcpkg_install_meson.cmake
[DEBUG] C:\Dev\vcpkg\scripts\cmake\vcpkg_install_meson.cmake has hash 715cf59e78124dcc0823803544dc12ca4ced4f46ebc43681f4f6cd23d3e99d92
[DEBUG] Trying to hash C:\Dev\vcpkg\scripts\cmake\vcpkg_install_msbuild.cmake
[DEBUG] C:\Dev\vcpkg\scripts\cmake\vcpkg_install_msbuild.cmake has hash d770323312cc635d641183bfa4a0c073f919c8a37b7d9b3576700b1e934f8cd4
[DEBUG] Trying to hash C:\Dev\vcpkg\scripts\cmake\vcpkg_install_nmake.cmake
[DEBUG] C:\Dev\vcpkg\scripts\cmake\vcpkg_install_nmake.cmake has hash e5540f04f33bc8eee3417da64e2c9a9aa2bc3dee62031238d7eb795ec31cb0b7
[DEBUG] Trying to hash C:\Dev\vcpkg\scripts\cmake\vcpkg_install_qmake.cmake
[DEBUG] C:\Dev\vcpkg\scripts\cmake\vcpkg_install_qmake.cmake has hash e9eedf4e8a942dbae43e25d0f7a38a8c3afe7761c5d9e042d1a52017e3dcebe6
[DEBUG] Trying to hash C:\Dev\vcpkg\scripts\cmake\vcpkg_list.cmake
[DEBUG] C:\Dev\vcpkg\scripts\cmake\vcpkg_list.cmake has hash f5de3ebcbc40a4db90622ade9aca918e2cf404dc0d91342fcde457d730e6fa29
[DEBUG] Trying to hash C:\Dev\vcpkg\scripts\cmake\vcpkg_minimum_required.cmake
[DEBUG] C:\Dev\vcpkg\scripts\cmake\vcpkg_minimum_required.cmake has hash 4da8ab803b83c8f1c7b6272298108b9f62a7cd0618e563242c8e584692534171
[DEBUG] Trying to hash C:\Dev\vcpkg\scripts\cmake\vcpkg_replace_string.cmake
[DEBUG] C:\Dev\vcpkg\scripts\cmake\vcpkg_replace_string.cmake has hash d43c8699ce27e25d47367c970d1c546f6bc36b6df8fb0be0c3986eb5830bd4f1
[DEBUG] Trying to hash C:\Dev\vcpkg\scripts\cmake\vcpkg_test_cmake.cmake
[DEBUG] C:\Dev\vcpkg\scripts\cmake\vcpkg_test_cmake.cmake has hash 8117a509de7f30dd5c1f4c35134c1b3180c0825ea03480aa4625e98ddd5476e6
[DEBUG] Trying to hash C:\Dev\vcpkg\scripts\cmake\z_vcpkg_apply_patches.cmake
[DEBUG] C:\Dev\vcpkg\scripts\cmake\z_vcpkg_apply_patches.cmake has hash aed81b9cb2c44e2736e6e912445044a2f0512c041ecbb72ffb3154e44b0044ba
[DEBUG] Trying to hash C:\Dev\vcpkg\scripts\cmake\z_vcpkg_escape_regex_control_characters.cmake
[DEBUG] C:\Dev\vcpkg\scripts\cmake\z_vcpkg_escape_regex_control_characters.cmake has hash 439c52c6ea37cecad4474528fb3b8019f2538380b3771d986e3d7811d8e274b5
[DEBUG] Trying to hash C:\Dev\vcpkg\scripts\cmake\z_vcpkg_fixup_rpath.cmake
[DEBUG] C:\Dev\vcpkg\scripts\cmake\z_vcpkg_fixup_rpath.cmake has hash 9efca0dc9594c2c2f094444cfbdc777cdcb9639a01435628d551127f4f4d735e
[DEBUG] Trying to hash C:\Dev\vcpkg\scripts\cmake\z_vcpkg_forward_output_variable.cmake
[DEBUG] C:\Dev\vcpkg\scripts\cmake\z_vcpkg_forward_output_variable.cmake has hash ae3440b77321516f29f0b51e26b8ecb05fd31f63d33949dc406a502c300b04ff
[DEBUG] Trying to hash C:\Dev\vcpkg\scripts\cmake\z_vcpkg_function_arguments.cmake
[DEBUG] C:\Dev\vcpkg\scripts\cmake\z_vcpkg_function_arguments.cmake has hash 11721a8a42a2aa2f6f285d485b21aed55834c2029140661bd8e2f0b709288d77
[DEBUG] Trying to hash C:\Dev\vcpkg\scripts\cmake\z_vcpkg_get_cmake_vars.cmake
[DEBUG] C:\Dev\vcpkg\scripts\cmake\z_vcpkg_get_cmake_vars.cmake has hash ce74da978727f15cc4eab6fa8a9c27023f0b51ef654cfbf7df8f3409fb950c43
[DEBUG] Trying to hash C:\Dev\vcpkg\scripts\cmake\z_vcpkg_prettify_command_line.cmake
[DEBUG] C:\Dev\vcpkg\scripts\cmake\z_vcpkg_prettify_command_line.cmake has hash 97e4e92ac6e9da544fc7b9a6695e0d491c2c095ec265d8c0a90bcb8168873ca3
[DEBUG] Trying to hash C:\Dev\vcpkg\scripts\cmake\z_vcpkg_setup_pkgconfig_path.cmake
[DEBUG] C:\Dev\vcpkg\scripts\cmake\z_vcpkg_setup_pkgconfig_path.cmake has hash 9bdbca9a026715b07908bca9786799b200c704ae826e7eb52f8bdcd5ca697254
[DEBUG] Trying to hash C:\Dev\vcpkg\scripts\ports.cmake
[DEBUG] C:\Dev\vcpkg\scripts\ports.cmake has hash 5a8e00cedff0c898b1f90f7d129329d0288801bc9056562b039698caf31ff3f3
[DEBUG] <abientries for zlib:x64-windows>
[DEBUG]   0001-Prevent-invalid-inclusions-when-HAVE_-is-set-to-0.patch|750b9542cb55e6328cca01d3ca997f1373b9530afa95e04213168676936e7bfa
[DEBUG]   0002-skip-building-examples.patch|835ddecfed752e0f49be9b0f8ff7ba76541cb0a150044327316e22ca84f8d0c2
[DEBUG]   0003-build-static-or-shared-not-both.patch|d6026271dcb3d8fc74b41e235620ae31576a798e77aa411c3af8cd9e948c02b1
[DEBUG]   0004-android-and-mingw-fixes.patch|37a43eddbcb1b7dde49e7659ae895dfd0ff1df66666c1371ba7d5bfc49d8b438
[DEBUG]   cmake|3.27.1
[DEBUG]   features|core
[DEBUG]   portfile.cmake|ac63047b644fa758860dd7ba48ff9a13b058c6f240b8e8d675b8fbba035976be
[DEBUG]   ports.cmake|5a8e00cedff0c898b1f90f7d129329d0288801bc9056562b039698caf31ff3f3
[DEBUG]   post_build_checks|2
[DEBUG]   powershell|7.2.11
[DEBUG]   triplet|x64-windows
[DEBUG]   triplet_abi|4556164a2cd3dd6f4742101eabb46def7e71b6e5856faa88e5d005aac12a803c-e36df1c7f50ab25f9c182fa927d06c19ae082e0d599f132b3f655784b49e4b33-f4a88a8e9062ec73eb4a671e8bb00c783a16b73c
[DEBUG]   usage|be22662327df993eebc437495add75acb365ab18d37c7e5de735d4ea4f5d3083
[DEBUG]   vcpkg-cmake|d40ba79eaf5eb0363c0ffec2e44072a1a0edf5db6653cfac5823ab15778a2d1b
[DEBUG]   vcpkg-cmake-wrapper.cmake|5d49ef2ee6448479c2aad0e5f732e2676eaba0411860f9bebabe6002d66f57d1
[DEBUG]   vcpkg.json|bc94e2540efabe36130a806381a001c57194e7de67454ab7ff1e30aa15e6ce23
[DEBUG]   vcpkg_copy_pdbs|d57e4f196c82dc562a9968c6155073094513c31e2de475694143d3aa47954b1c
[DEBUG]   vcpkg_fixup_pkgconfig|588d833ff057d3ca99c14616c7ecfb5948b5e2a9e4fc02517dceb8b803473457
[DEBUG]   vcpkg_from_git|8f27bff0d01c6d15a3e691758df52bfbb0b1b929da45c4ebba02ef76b54b1881
[DEBUG]   vcpkg_from_github|b743742296a114ea1b18ae99672e02f142c4eb2bef7f57d36c038bedbfb0502f
[DEBUG]   vcpkg_replace_string|d43c8699ce27e25d47367c970d1c546f6bc36b6df8fb0be0c3986eb5830bd4f1
[DEBUG] </abientries>
[DEBUG] Default binary cache path is: C:\Users\bion\AppData\Local\vcpkg\archives
Restored 0 package(s) from C:\Users\bion\AppData\Local\vcpkg\archives in 551 us. Use --debug to see more details.
Installing 1/1 zlib:x64-windows...
Building zlib:x64-windows...
[DEBUG] Detecting host with IsWow64Process2
[DEBUG] 1007: CreateProcessW("C:\Dev\vcpkg-downloads\tools\cmake-3.27.1-windows\cmake-3.27.1-windows-i386/bin/cmake.exe" -DALL_FEATURES= -DCURRENT_PORT_DIR=C:/Dev/vcpkg/ports/zlib -D_HOST_TRIPLET=x64-windows -DFEATURES=core -DPORT=zlib -DVERSION=1.2.13 -DVCPKG_USE_HEAD_VERSION=0 -D_VCPKG_DOWNLOAD_TOOL=BUILT_IN -D_VCPKG_EDITABLE=0 -D_VCPKG_NO_DOWNLOADS=0 -DZ_VCPKG_CHAINLOAD_TOOLCHAIN_FILE=C:/Dev/vcpkg/scripts/toolchains/windows.cmake --debugger "--debugger-pipe=\\.\pipe\tespipe" -DCMD=BUILD -DDOWNLOADS=C:/Dev/vcpkg-downloads -DTARGET_TRIPLET=x64-windows -DTARGET_TRIPLET_FILE=C:/Dev/vcpkg/triplets/x64-windows.cmake -DVCPKG_BASE_VERSION=2999-12-31 -DVCPKG_CONCURRENCY=65 -DVCPKG_PLATFORM_TOOLSET=v143 "-DGIT=C:/Program Files/Git/cmd/git.exe" "-DVCPKG_PORT_CONFIGS=C:\Dev\vcpkg\installed\x64-windows\share\vcpkg-cmake\vcpkg-port-config.cmake" -DVCPKG_ROOT_DIR=C:/Dev/vcpkg -DPACKAGES_DIR=C:/Dev/vcpkg/packages -DBUILDTREES_DIR=C:/Dev/vcpkg/buildtrees -D_VCPKG_INSTALLED_DIR=C:/Dev/vcpkg/installed -DDOWNLOADS=C:/Dev/vcpkg-downloads -DVCPKG_MANIFEST_INSTALL=OFF -P "C:\Dev\vcpkg\scripts\ports.cmake")
Running with debugger on.
Waiting for debugger client to connect...
```

And confirming the command line at the Windows level with Process Monitor output:

>Parent PID:	41416
>Command line:	"C:\Dev\vcpkg-downloads\tools\cmake-3.27.1-windows\cmake-3.27.1-windows-i386/bin/cmake.exe" -DALL_FEATURES= -DCURRENT_PORT_DIR=C:/Dev/vcpkg/ports/zlib -D_HOST_TRIPLET=x64-windows -DFEATURES=core -DPORT=zlib -DVERSION=1.2.13 -DVCPKG_USE_HEAD_VERSION=0 -D_VCPKG_DOWNLOAD_TOOL=BUILT_IN -D_VCPKG_EDITABLE=0 -D_VCPKG_NO_DOWNLOADS=0 -DZ_VCPKG_CHAINLOAD_TOOLCHAIN_FILE=C:/Dev/vcpkg/scripts/toolchains/windows.cmake **--debugger "--debugger-pipe=\\.\pipe\tespipe"** -DCMD=BUILD -DDOWNLOADS=C:/Dev/vcpkg-downloads -DTARGET_TRIPLET=x64-windows -DTARGET_TRIPLET_FILE=C:/Dev/vcpkg/triplets/x64-windows.cmake -DVCPKG_BASE_VERSION=2999-12-31 -DVCPKG_CONCURRENCY=65 -DVCPKG_PLATFORM_TOOLSET=v143 "-DGIT=C:/Program Files/Git/cmd/git.exe" "-DVCPKG_PORT_CONFIGS=C:\Dev\vcpkg\installed\x64-windows\share\vcpkg-cmake\vcpkg-port-config.cmake" -DVCPKG_ROOT_DIR=C:/Dev/vcpkg -DPACKAGES_DIR=C:/Dev/vcpkg/packages -DBUILDTREES_DIR=C:/Dev/vcpkg/buildtrees -D_VCPKG_INSTALLED_DIR=C:/Dev/vcpkg/installed -DDOWNLOADS=C:/Dev/vcpkg-downloads -DVCPKG_MANIFEST_INSTALL=OFF -P "C:\Dev\vcpkg\scripts\ports.cmake"
>Current directory:	C:\Dev\vcpkg\

Reply:
Ok, the issue is not the parsing of the command line but the shell you use for the command. I typically run in git bash and that eats the `\` while cmd does not. I am somehow inclined to allow forward slashes and let them be transform into backslashes.